src/Components/Tabs/Tabs.js

```diff
@@ -115,17 +116,20 @@ class Tabs extends Component {
   }
 
   getTabWidth(width) {
-    const { scrollEnabled, actionItems } = this.props;
+    const { scrollEnabled, actionItems, fixedTabWidth } = this.props;
+    const hasFixedTabWidth = !!fixedTabWidth;
+    let fixedWidth;
 
     if (!scrollEnabled) {
       let tabWidth = width / actionItems.length;
+      fixedWidth = fixedWidth * actionItems.length;
 
       this.setState({
-        tabWidth: tabWidth,
+        tabWidth: hasFixedTabWidth ? fixedTabWidth : tabWidth,
```

Comment:
You calculated `fixedWidth` above but then you're not using it here. Is that right?